src/overlays/actors/ovl_En_Gakufu/z_en_gakufu.c

```diff
@@ -34,28 +39,253 @@ const ActorInit En_Gakufu_InitVars = {
     (ActorFunc)EnGakufu_Draw,
 };
 
-#endif
+Vec3f sRewardDropsSpawnPos = {
+    -710.0f,
+    -123.0f,
+    -3528.0f,
+};
+
+u8 sRewardDropsIndex[] = {
+    3,  // CLOCK_TIME(0, 0) - CLOCKTIME(1, 0)
+    12, // CLOCK_TIME(1, 0) - CLOCKTIME(2, 0)
+    6,  // CLOCK_TIME(2, 0) - CLOCKTIME(3, 0)
+    12, // CLOCK_TIME(3, 0) - CLOCKTIME(4, 0)
+    9,  // CLOCK_TIME(4, 0) - CLOCKTIME(5, 0)
+    12, // CLOCK_TIME(5, 0) - CLOCKTIME(6, 0)
+    0,  // CLOCK_TIME(6, 0) - CLOCKTIME(7, 0)
+    12, // CLOCK_TIME(7, 0) - CLOCKTIME(8, 0)
+    3,  // CLOCK_TIME(8, 0) - CLOCKTIME(9, 0)
+    12, // CLOCK_TIME(9, 0) - CLOCKTIME(10, 0)
+    6,  // CLOCK_TIME(10, 0) - CLOCKTIME(11, 0)
+    12, // CLOCK_TIME(11, 0) - CLOCKTIME(12, 0)
+    3,  // CLOCK_TIME(12, 0) - CLOCKTIME(13, 0)
+    12, // CLOCK_TIME(13, 0) - CLOCKTIME(14, 0)
+    6,  // CLOCK_TIME(14, 0) - CLOCKTIME(15, 0)
+    12, // CLOCK_TIME(15, 0) - CLOCKTIME(16, 0)
+    3,  // CLOCK_TIME(16, 0) - CLOCKTIME(17, 0)
+    12, // CLOCK_TIME(17, 0) - CLOCKTIME(18, 0)
+    6,  // CLOCK_TIME(18, 0) - CLOCKTIME(19, 0)
+    12, // CLOCK_TIME(19, 0) - CLOCKTIME(20, 0)
+    9,  // CLOCK_TIME(20, 0) - CLOCKTIME(21, 0)
+    12, // CLOCK_TIME(21, 0) - CLOCKTIME(22, 0)
+    6,  // CLOCK_TIME(22, 0) - CLOCKTIME(23, 0)
+    12, // CLOCK_TIME(23, 0) - CLOCKTIME(0, 0)
+};
```

Comment:
The comments are supposed to be ranges of time? Because I first though they were subtractions and I was kinda confused.
Maybe use `~` instead of `-`?

Reply:
Yeah these are ranges! I can try `~` or maybe even the word `to`